src/components/thread/messages/human.tsx

```diff
@@ -58,6 +58,7 @@ export function HumanMessage({
       {
         checkpoint: parentCheckpoint,
         streamMode: ["values"],
+        streamSubgraphs: true,
```

Comment:
`streamResumable` here too?
```suggestion
        streamSubgraphs: true,
        streamResumable: true,
```